js/jquery-accessibleMegaMenu.js

```diff
@@ -262,28 +270,39 @@ limitations under the License.
                     menu.find('[aria-expanded=true]')
                         .attr('aria-expanded', 'false')
                         .removeClass(settings.openClass)
-                        .filter('.' + settings.panelClass)
+                        .closest('.' + settings.panelClass)
+                        .removeClass(settings.openClass)
                         .attr('aria-hidden', 'true');
                 }
-            } else {
+            }
+            // Toggle panels.
+            else {
                 clearTimeout(that.focusTimeoutID);
-                topli.siblings()
-                    .find('[aria-expanded]')
-                    .attr('aria-expanded', 'false')
-                    .removeClass(settings.openClass)
-                    .filter('.' + settings.panelClass)
-                    .attr('aria-hidden', 'true');
+                // Close previously open top level link and its panel.
+                var openli = menu.find('[aria-expanded=true]').parent();
+                if (!openli.is(topli)) {
+                    openli.find('[aria-expanded]')
+                        .attr('aria-expanded', 'false')
+                        .removeClass(settings.openClass)
+                        .closest('.' + settings.panelClass)
+                        .removeClass(settings.openClass)
+                        .attr('aria-hidden', 'true');
+                }
+                // Open current top level link and its panel.
                 topli.find('[aria-expanded]')
                     .attr('aria-expanded', 'true')
                     .addClass(settings.openClass)
-                    .filter('.' + settings.panelClass)
+                    .closest('.' + settings.panelClass)
```

Comment:
.closest() doesn't trigger the addClass or aria-hidden toggle for me
changing this to .siblings() works

Reply:
changing .closest() to .siblings() in all instances where settings.panelClass is adding/removing settings.openClass and toggling aria-hidden fixes this